src/cljs/main/broadfcui/common/table.cljs

```diff
@@ -197,7 +203,9 @@
    (let [restored
          (persistence/try-restore
           {:key (:state-key props)
-           :validator (fn [stored-value] (= (set (keys stored-value)) persistence-keys))
+           :validator (fn [stored-value]
+                        (or (not (:v props))
+                            (= (:v props) (:v stored-value))))
```

Comment:
do you still need the `(= (set (keys stored-value)) persistence-keys)` for those that do not have a `:v`?